examples/tutorial/07_Geographic_Data.ipynb

```diff
@@ -113,6 +113,7 @@
    "metadata": {},
    "source": [
     "It is often useful to project the data ahead of time, particularly for large datasets, because when working interactively we often display the data many times and don't want to reproject it every time.\n",
+    "> <div style=\"color:red;\">UNCLEAR: how would one go about using already projected points in some overlay?</div>\n",
```

Comment:
```suggestion
```

Reply:
I certainly would not want an editorial comment to be added; however, I think an example here would be useful:
I for one do not know how to write one based on the suggestions in the text!

Reply:
See the link I added to the previous sentence.